src/scenes/Wallet/components/ActionButtonHeader/index.js

```diff
@@ -1,15 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 import { injectIntl, defineMessages } from 'react-intl';
+import { inject, observer } from 'mobx-react';
 import { withStyles, Button, Tooltip } from '@material-ui/core';
-
 import EncryptDialog from '../EncryptDialog';
 import EncryptStatusDialog from '../EncryptStatusDialog';
 import RestoreWalletDialog from '../RestoreWalletDialog';
 import ChangePassphraseDialog from '../ChangPassphraseDialog';
 import ChangePassphraseStatusDialog from '../ChangePassphraseStatusDialog';
-import appActions from '../../../../redux/App/actions';
+import ErrorDialog from '../../../../components/ErrorDialog';
```

Comment:
This should not be necessary. The `ErrorDialog` is already in `App/index.js` which is a top level component that should handle this. All you have to do is set the `error` in the mobx store and it should show up. This is duplicating the dialog.